### PR TITLE
Add an Explanatory Error Message for uint Types

### PR DESCRIPTION
**Pull request by alexbarghi-nv**
Clarify that unsigned integer types are not supported as vertex ids in the error message.  Resolves issue reported by @ogreen.

### CODE REVIEW

File: python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py [hunk 1]
```diff
@@ -255,6 +255,11 @@ def __from_edgelist(
             elif elist[source].dtype not in [np.int32, np.int64] or elist[
                 destination
             ].dtype not in [np.int32, np.int64]:
+                if elist[destination].dtype in [np.uint32, np.uint64]:
```
Comment (rlratzel) [inline python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py:258]: should this also check `elist[source].dtype` ?

Comment (alexbarghi-nv) [reply python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py]: fixed

**Comment by alexbarghi-nv** [comment]
@rlratzel can you re-review?  All changes should be addressed now.

**Pull request merged**